stable/SC-System-and-Communications-Protection/policy-scc.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: policy.open-cluster-management.io/v1
 kind: Policy
 metadata:
-  name: policy-securitycontextconstraints
+  name: policy-sc-constraint
```

I think I'd prefer using its short name, `scc`. It'd match the filename also.
```suggestion
  name: policy-scc
```

---

Great!